specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/webapplicationfirewall.json

```diff
@@ -519,6 +519,13 @@
               "description": "The rules that are applied to the logs for scrubbing."
             }
           }
+        },
+        "JSChallengeCookieExpirationInMins": {
```

Comment:
>JSChallengeCookieExpirationInMins [](http://example.com/codeflow?start=9&length=33)

I think preferred camelCase rules would make js lower case in this name. Consider naming this jsChallengeCookieExpirationMinutes

Reply:
done